### PR TITLE
functionals

### PR DESCRIPTION
**Pull request by paocorrales**
**p. 211** The editor ask to close the space between "sea" and the question mark. I think the she can't see the emojis on her pdf viewer. 

**p. 223** The editor ask to describe a table in the text. This is a major change on the paragraph, do you want me to do write it?

### CODE REVIEW

**Comment by hadley** [comment]
Which table?

**Comment by paocorrales** [comment]
I've put an inline comment above the table (line 487)!

File: Functionals.Rmd [hunk 1]
```diff
@@ -485,7 +484,7 @@ There are 23 primary variants of `map()`. So far, you've learned about five (`ma
 * Iterate over any number of inputs with `pmap()`.
 
 The map family of functions has orthogonal input and outputs, meaning that we can organise all the family into a matrix, with inputs in the rows and outputs in the columns. Once you've mastered the idea in a row, you can combine it with any column; once you've mastered the idea in a column, you can combine it with any row. 
-
+<!-- I'm referring to this table. The editor thinks that it has to be explained in the text. -->
```
Comment (hadley) [inline Functionals.Rmd]: Maybe just: That relationship is summarised in the following table:

File: Functionals.Rmd [hunk 2]
```diff
@@ -21,10 +21,10 @@ cols("#8DD3FB")
 ## Introduction
 \index{functionals}
 
-> "To become significantly more reliable, code must become more transparent.
+> _To become significantly more reliable, code must become more transparent.
```
Comment (hadley) [inline Functionals.Rmd]: I think the `_` can go now that @eliocamp tweaked the tex

**Comment by paocorrales** [comment]
Yes, of course. I already talk to Elio about this change!

**Comment by hadley** [comment]
Thanks!

**Pull request merged**